### PR TITLE
Brig Physician, Lawyer, Clerk, Psychiatrist all need 20 pop to play now.

### PR DESCRIPTION
**Pull request by JamieD1**
If you are playing a job that requires other people, you should need other people..

Also stops no sec brig physician, mostly being security.

:cl:  
tweak: Brig Physician, Lawyer, Clerk, Psychiatrist all need 20 pop to play now.
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Damn I love playing Psychologist even on lowpop ):

**Comment by Anvilman6** [comment]
A more elegant solution (if possible) would be to require a Sec Officer/Warden/HoS to be able to play Brig Physician or Lawyer, since general population levels aren't necessarily indicative of them being impactful.

I suppose I can see where you're coming from with Clerk and Psych, though honestly I think Clerk can work with any population (it just works better with higher ones).

**Comment by JamieD1** [comment]
> A more elegant solution (if possible) would be to require a Sec Officer/Warden/HoS to be able to play Brig Physician or Lawyer, since general population levels aren't necessarily indicative of them being impactful.
> 
> I suppose I can see where you're coming from with Clerk and Psych, though honestly I think Clerk can work with any population (it just works better with higher ones).

Could be done, however it would make the job selection code more complicated.

**Comment by Tipy1802** [comment]
I don't see the point, even in bellow 20 pop there are still people who those jobs may interact with. All things considered those jobs don't get much interaction on high pop as well as they are gimmicky roleplay jobs.

> Also stops no sec brig physician, mostly being security.

I mean a brig physician being security on low pop is not bad. It's preferable to having no security or having the hoard of greytiders and the jobs with powergaming loot validhunt antags on no sec rounds

File: code/controllers/subsystem/job.dm [hunk 1]
```diff
@@ -582,6 +583,11 @@ SUBSYSTEM_DEF(job)
 			J.total_positions = officer_positions
 			J.spawn_positions = officer_positions
 
+	var/datum/job/phy = SSjob.GetJob("Brig Physician")
+	if(J.total_positions > 1)
+		phy.total_positions = 1
+		phy.spawn_positions = 1
```
Comment (adamsong) [inline code/controllers/subsystem/job.dm]: Why is this here? It just unnecessarily makes it impossible to have more than 1 brig phys, which currently won't happen outside of bus.

File: code/controllers/subsystem/job.dm [hunk 2]
```diff
@@ -582,6 +583,11 @@ SUBSYSTEM_DEF(job)
...
+
```
Comment (JamieD1) [inline code/controllers/subsystem/job.dm]:
```suggestion
```
And this is why we have maintainers...

**Comment by JamieD1** [comment]
> I don't see the point, even in bellow 20 pop there are still people who those jobs may interact with. All things considered those jobs don't get much interaction on high pop as well as they are gimmicky roleplay jobs.
> 
> > Also stops no sec brig physician, mostly being security.
> 
> I mean a brig physician being security on low pop is not bad. It's preferable to having no security or having the hoard of greytiders and the jobs with powergaming loot validhunt antags on no sec rounds

They are medical.. not sec..

**Comment by Hopekz** [comment]
> They are medical.. not sec..

Alright, well, let us stick them back in medical, then

**Comment by Tipy1802** [comment]
> > I don't see the point, even in bellow 20 pop there are still people who those jobs may interact with. All things considered those jobs don't get much interaction on high pop as well as they are gimmicky roleplay jobs.
> > > Also stops no sec brig physician, mostly being security.
> > 
> > 
> > I mean a brig physician being security on low pop is not bad. It's preferable to having no security or having the hoard of greytiders and the jobs with powergaming loot validhunt antags on no sec rounds
> 
> They are medical.. not sec..

yeah but it's lowpop with no sec, literally anyone is allowed to validhunt anyways

**Review by Bibby0110 — APPROVED** [review]
